scripts/variantreport_p_sample.rmd

```diff
@@ -180,6 +180,16 @@ getIdentCols <- function (dataframe) {
     }
     return(cols_new)
 }
+
+dedupeMuts <- function (mut, sigmut.df, dedup.df) {
+        # get variant group per mutation
+        duped_muts <- grep (mut, sigmut.df$value)
+        grouped <- sigmut.df$name[duped_muts]
+        groupName <- paste(grouped, collapse = ",")
+      
+        dedup.df$name[dedup.df$value == mut] <- groupName
+        return (dedup.df)
+}  
```

Comment:
Is this procedure a duplicate of what has already landed in `main`?

Reply:
I'm not sure what you mean.

Reply:
https://github.com/BIMSBbioinfo/pigx_sarscov2_ww/blob/main/scripts/report_scripts/variantreport_p_sample.Rmd#L149

Reply:
Ah, only see now that it's tracked as added here. 
This is probably a result of me not properly pulling and rebasing. 

So yes, it's a duplicate. But I will probably just resolve this when I rebase I guess.